+++
author = "Kevin Shoemaker"
categories = ["Population modeling", "Central grasslands ecosystem", "Wildlife Management", "Projects", "Posts"]
date = "2017-01-17"
description = "Modeling evolution of plague resistance in prairie dogs"
featured = "pdog2.jpg"
featuredalt = ""
featuredpath = "/img"
linktitle = "Pdog"
title = "Understanding the potential for widespread plague resistance in prairie dogs"
type = "post"
+++

Prairie dogs are keystone species of North Americaâ€™s central grasslands, and have experienced dramatic population reductions over the past century due to sylvatic plague, a highly virulent disease introduced to North America ca. 1900. The APE lab is involved in building a novel host-resistance modeling framework to investigate the genetic, demographic and environmental conditions that promote or impede resistance to plague and other similar virulent pathogens in complex and dynamic ecosystems. The prairie dog-plague system provides an excellent model to understand the conditions under which evolutionary rescue occurs in complex and dynamic ecological systems, and will help illuminate a novel and potential large-scale conservation approach to address 1) the biggest threat facing the prairie dog ecosystem that spans from Canada to Mexico - plague and 2) species around the world threatened with novel diseases and climate change.

## A novel modeling framework for understanding host resistance in complex real-world landscapes

We will use a flexible lattice population structure in which each node contains a stage-structured host population, allowing integration of spatial raster data of environmental variables and enabling multiple modes of dispersal and disease transmission. Natural selection will be modeled as vital rate reductions contingent upon population-level genetic makeup. Selection pressures will emerge from a statistical epidemiological model of disease incidence and spread. Resistance-conferring, diploid gene loci will be modeled with arbitrary numbers of genes, allelic variants, dominance of resistance, gene interactions (epistasis; we will not model chromosomal linkage) and alternative modes of resistance (i.e., alternative sets of genes/alleles that confer resistance). Coevolution will be modeled as a change in virulence over time. This modeling framework will be written in R/C++.

![Schematic of plague model](/img/Plague Model Schematic1.png)

### Some example videos...

**Unstructured landscape**:    
- https://youtu.be/45_VITXJgJs   
- https://youtu.be/8PhD-aDqTj4   
- https://youtu.be/NaCT5NDUwTo   

**Structured landscape**:     
- https://youtu.be/fWb79btL8G0    
- https://youtu.be/MFeZLlB0xV8    
- https://youtu.be/TF9fVWDkfFA    

Modeling lead: [Kevin Shoemaker](http://naes.unr.edu/shoemaker/)
Overall project lead: [Ana Davidson, UNM](http://anadavidson.weebly.com/)

Key collaborators:
[Loren Cassin Sackett, USF](http://labs.cas.usf.edu/sackett/Sackett_Lab/People.html)
[Joseph Busch, NAU](http://www.mggen.nau.edu/)
[Katherine Richgels, USGS](https://www.nwhc.usgs.gov/staff/katie_richgels.jsp)
[Tonie Rocke, USGS](https://www.nwhc.usgs.gov/staff/tonie_rocke.jsp)
[Elizabeth Hunter, UNR APE lab](http://elizabethhunter.weebly.com/)
